go.mod

```diff
@@ -7,5 +7,4 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/viper v1.7.1
-	golang.org/x/lint v0.0.0-20200302205851-738671d3881b // indirect
```

Comment:
Remove it here as well please: https://github.com/tome-sh/tome/blob/master/.github/workflows/go.yaml#L29